frontend/src/sass/base/_variables.scss

```diff
@@ -1,14 +1,14 @@
 @import '~base/sass/base/functions';
 
-$rem-base-size: 16 !default;
+$rem-base-size: 10 !default;
```

Comment:
Will this break when the community uses them on vanilla Open edX?

Reply:
@OmarIthawi If this does break for vanilla, I'll work out an interim solution. Not for now, but for later, is there a reason why an external css that defines the `rem-base-size`? from which Figures uses. I mean, this is part of the cascade in CSS, right? That and we really want to move away from building the assets and including them in the repo, instead building the assets into the deployable python bundle we push to PyPI

until we automate our build/deploy to PyPI and th

Reply:
Thanks @johnbaldwin! I did use the wrong word here. I don't think it will "break break", and [Matej explained that this is a temporary solution](https://appsembler.slack.com/archives/C4FHDM4U9/p1552848244002800?thread_ts=1552757366.001000&cid=C4FHDM4U9), which I think is OK.

I simply worried that the font will look very small on Open edX deployments to the point that forces community members to fork and recompile, which is anti-opensource.

> we really want to move away from building the assets and including them in the repo, instead building the assets into the deployable python bundle we push to PyPI

Yes please! I really like that you're working on keeping PyPI as the no. 1 installation option.

> If this does break for vanilla, I'll work out an interim solution. Not for now, but for later, is there a reason why an external css that defines the rem-base-size?

Yes, `rem` is all about cascade/inheritance as far as I know. @grozdanowski knows better than I do, but I think it's more like saying `font-size: 100%`. It's not just cascaded, it is also depends on what the parent `font-size` value is.

Reply:
@johnbaldwin @OmarIthawi To clarify - there's no need to fork. One can pass on env variables and rebuild on the fly. This is also a convenient tool for other purposes (if someone wants to brand their Figures intance for themselves using font sizes or colours).
And yes - it is an issue in regards to cascading nature of CSS - `rem` is defined by the first ever definition of font size in the rendered document.